src/styles/_base.css

```diff
@@ -12,15 +16,50 @@ body {
   margin: 0;
 }
 
+main {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 375px);
```

Comment:
Why `375`?

Reply:
Please add a comment to your code that explains the number (if you are still sure you want to use it).